index.html

```diff
@@ -0,0 +1,175 @@
+            margin-left: 30%;
+            margin-top: 60px;
+        }
+
+        article.women div {
+            margin-left: 5%;
+            margin-top: 60px;
+        }
+
+        .new-models {
+            margin: 100px auto;
+            text-align: center;
+        }
+
+        .lower-block {
+            background-color: #97DA8C;
+            height: 350px;
+            width: 1110px;
+            margin-left: auto;
+            margin-right: auto;
+            margin-bottom: 200px;
+        }
+
+        footer {
+            background-color: black;
+            height: 363px;
+            padding-top: 100px;
+            padding-left: 600px;
+            box-sizing: border-box;
+            margin-bottom: 20px;
+        } 
+
+        footer > div {
+            height: 180px;
+            width: 206px;
+            display: inline-block;
+            vertical-align: top; 
+        }
+
+        footer div div {
+            margin-left: 10px;
+            margin-top: 5px;
+            color: #CBCFD4;
+        }
+
+        footer div div:first-child {
+            color: white;
+            margin-bottom: 10px;
+        }
+
+        .copyright {
+            display: inline-block;
+            margin-left: 400px;
+            margin-bottom: 20px;
+        }
+    </style>
+</head>
+<body>
+    <nav>
+        <span class="for-who">Для мужчин</span>
+        <span class="for-who">Для женщин</span>
+        <span class="for-who">Для детей</span>
+        <span>Оплата</span>
+        <span>Доставка</span>
+    </nav>
+    <div class="upper-ad"></div>
+    <section>
+        <div>БЕСПЛАТНАЯ ДОСТАВКА ОТ 3000Р</div>
+        <div>Сделайте заказ и получите подарок!</div>
+    </section>
+    <article class="men">
+        <div>МУЖСКАЯ КОЛЛЕКЦИЯ</div>
+        <div>Обувь, полуверы, костюмы</div>
+    </article>
+    <article class="women">
+        <div>ЖЕНСКАЯ КОЛЛЕКЦИЯ</div>
+        <div>Обувь, полуверы, костюмы</div>
+    </article>
+    <div class="new-models">НОВЫЕ МОДЕЛИ</div>
+    <div class="lower-block"></div>
+    <footer>
+        <div>
+            <div>Основные ссылки</div>
+            <div>О компании</div>
+            <div>Каталог</div>
+            <div>Доставка</div>
+            <div>Оплата</div>
+        </div>
+        <div>
+            <div>Категории</div>
+            <div>Мужская одежда</div>
+            <div>Женская одежда</div>
+            <div>Детская одежда</div>
+            <div>Для животных</div>
+        </div>
+        <div>
+            <div>ПОЛЕЗНЫЕ ССЫЛКИ</div>
+            <div>Таблица размеров</div>
+            <div>Блог о моде</div>
+            <div>Наша миссия</div>
+        </div>
+        <div>
+            <div>СОЦ СЕТИ</div>
+            <div>VK.com</div>
+             <div>Facebook</div>
+            <div>Instagram</div>
+        </div>
+    </footer>
+    <div class="copyright">Copyright © 2019. Все права защищены</div>
+    <div class="copyright">Политика приватности</div>
+</body>
+</html>
```

Comment:
Forgot to mention on `git` lecture: always add empty line to the bottom of each file.
The reason for that is lying in Unix system text files structure history, so just always stick to this rule - git likes it!